### PR TITLE
Convert whats_left into a CPython script

### PR DESCRIPTION
**Pull request by verhovsky**
so that we can parse command line options to configure what to print and how to print it.

### CODE REVIEW

File: whats_left.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
 # It's recommended to run this with `python3 -I not_impl_gen.py`, to make sure
```
Comment (fanninpm) [inline whats_left.py:3]:
```suggestion
# It's recommended to run this with `python3 -I whats_left.py`, to make sure
```

**Comment by youknowone** [comment]
oh, i liked this patch but it was forgotten. @verhovsky could you please open it again?

**Comment by youknowone** [comment]
@fanninpm please review again?

**Review by fanninpm — APPROVED** [review]
This should make it easier for people running Windows.

**Comment by youknowone** [comment]
@verhovsky Thank you! and I am really sorry to forgetting this PR for long time

**Pull request merged**